tests/data/REFERENCE_PPP_o2_EcoliCore_ac.csv

```diff
@@ -0,0 +1,21 @@
+,EX_o2_LPAREN_e_RPAREN_,objective_lower_bound,objective_upper_bound
```

Comment:
@KristianJensen this is now used as reference in `test_one_variable_sequential_metabolite` which previously used `REFERENCE_PPP_o2_EcoliCore`. Since this test changes the objective to a demand reaction for `o2_e` it did yield result identical to `REFERENCE_PPP_o2_EcoliCore` which has biomass as objective. I changed this to ac_e and made a new reference, could you please confirm/suggest changes